Automattic/wp-calypso
# Hosting Metrics: Apply new design to PHP and web server logs

Related to https://github.com/Automattic/dotcom-forge/issues/3392

## Proposed Changes

This PR updates the design of the `PHP logs` and `Webserver Logs` table to the design suggested by Javier in Figma: [Figma file](https://www.figma.com/file/RUPAbTTMYO7Eqxlas920Yb/Site-Logging?type=design&node-id=428-3350&mode=design&t=Vs5ABa4YHfFZzVcV-0)

**Before**

<img width="1214" alt="Screenshot 2023-08-28 at 3 22 15 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/868a541a-56fc-42d1-8b63-4f4b40c3b419">

**After**

<img width="1247" alt="Screenshot 2023-08-28 at 3 22 59 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/2f549aff-f105-496c-a9b2-a9260241ee2e">

**Mobile view before**

<img width="569" alt="Screenshot 2023-08-28 at 3 24 31 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/7894f884-20a4-48cd-922d-d0037cc7645d">

**Mobile view after**

<img width="343" alt="Screenshot 2023-08-28 at 3 24 54 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/b789eb51-446b-44e7-b7e7-e15f128fd774">

## Testing Instructions

* Make sure you have an Atomic site
* Navigate to `Tools > Site Monitoring`
* Click on `PHP Logs` tab and check that the new design is consistent with Figma design
* Click on `Webserver Logs` tab and check that the new design is consistent with Figma design
* Check the mobile view for both logs 
* Try using the navigation (pagination) at the bottom to make sure it works as expected
* Look for any weird behaviour in the UI

## Pre-merge Checklist
